onnxruntime/core/providers/nuphar/compiler/x86/scheduler/ort_type_schedule.cc

```diff
@@ -176,6 +245,7 @@ static Status ConvScheduleX86(const tvm::Tensor& tensor,
 
   ctx_sched.schedule[tensor->op].reorder({b, oc_chunk, y, xo, ic_chunk, m, n, ic_block, xi, oc_block});
 
+#if 0
```

Comment:
Do we keep ```#if 0``` here on purpose?

Reply:
OK will remove.

---
In reply to: [351534585](https://github.com/microsoft/onnxruntime/pull/2505#discussion_r351534585) [](ancestors = 351534585)